report_common_nrq/report/common_template.xml

```diff
@@ -354,6 +354,27 @@
             </tr>
         </table>
     </template>
+    <template id="price_unit_value_format">
+        <!--
+            It should define the price_unit value.
+            e.g.
+            <t t-set="price_unit" t-value="l.price_unit" />
```

Comment:
What is "l" in this template? This is not defined in the same template.
Maybe you use "l in doc.invoice_line_ids" in L383(before change), but this is defined in another template.